tiledb/group.py

```diff
@@ -22,7 +22,7 @@ class Group(CtxMixin, lt.Group):
 
     :param uri: The URI to the Group
     :type uri: str
-    :param mode: Read mode ('r') or write mode ('w')
+    :param mode: Read mode ('r'), write mode ('w'), or modify exclusive ('m')
```

Comment:
Maybe explicitly mention that `'m'` is for deletes, similar to [delete_fragments](https://tiledb-inc-tiledb.readthedocs-hosted.com/projects/tiledb-py/en/stable/python-api.html#tiledb.libtiledb.Array.delete_fragments)?